domain/credential/state/types.go

```diff
@@ -49,6 +49,17 @@ type Credential struct {
 	InvalidReason string `db:"invalid_reason"`
 }
 
+type CredentialAttribute struct {
```

Comment:
Comment for the type.

Reply:
Do these ever need exporting? They shouldn't really escape the state sub-packages.